src/Microsoft.DotNet.Cli.Utils/CommandResolution/MSBuildProject.cs

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.Build.Evaluation;
+using NuGet.Frameworks;
+using NuGet.ProjectModel;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    internal class MSBuildProject : IProject
+    {
+        private Project _project;
+
+        private string _msBuildExePath;
+
+        public string DepsJsonPath
+        {
+            get
+            {
+                return _project
+                    .AllEvaluatedProperties
+                    .FirstOrDefault(p => p.Name.Equals("_ProjectDepsFilePath"))
```

Comment:
Yes, we chatted about it during stand up. I was under the impression you were going to change it.